docs-site/docs/user/organization/groups/bot-accounts/bot-accounts-intro.md

```diff
@@ -4,4 +4,4 @@ id: introduction
 
 # Bot Accounts
 
-<!-- TODO: Bot accounts intro -->
+Bot accounts are to be used when performing actions which can't be attributed to a human user. For example, using an uploader.
```

Comment:
Remove the `to be` from here

Reply:
30ed197bcd870d4b31e3b30cf22454ef0bee829e